sample/src/main/java/io/ona/kujaku/sample/activities/PassiveRecordObjectActivity.java

```diff
@@ -0,0 +1,201 @@
+package io.ona.kujaku.sample.activities;
+
+import android.location.Location;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.ona.kujaku.callbacks.OnLocationServicesEnabledCallBack;
+import io.ona.kujaku.domain.Point;
+import io.ona.kujaku.listeners.TrackingServiceListener;
+import io.ona.kujaku.sample.R;
+import io.ona.kujaku.services.TrackingService;
+import io.ona.kujaku.services.configurations.TrackingServiceDefaultUIConfiguration;
+import io.ona.kujaku.services.options.TrackingServiceHighAccuracyOptions;
+import io.ona.kujaku.views.KujakuMapView;
+
+public class PassiveRecordObjectActivity extends BaseNavigationDrawerActivity implements TrackingServiceListener {
+
+    private static final String TAG = PassiveRecordObjectActivity.class.getName();
+
+    private KujakuMapView kujakuMapView;
+
+    private Button startStopBtn;
+    private Button forceLocationBtn;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        kujakuMapView = findViewById(R.id.kmv_passiveRecordObject_mapView);
+        kujakuMapView.onCreate(savedInstanceState);
+        kujakuMapView.showCurrentLocationBtn(true);
+
+        this.startStopBtn = findViewById(R.id.btn_passiveRecordObject_StartStopRecording);
+        this.forceLocationBtn = findViewById(R.id.btn_passiveRecordObject_ForcePoint);
+
+        this.startStopBtn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (((Button)v).getText().equals(getString(R.string.start_recording))) {
+
+                    // Start Service
+                    kujakuMapView.startTrackingService(getApplicationContext(),
+                            PassiveRecordObjectActivity.class,
+                            PassiveRecordObjectActivity.this,
+                            new TrackingServiceHighAccuracyOptions(),
+                            new TrackingServiceDefaultUIConfiguration());
+
+                    ((Button)v).setText(getString(R.string.stop_recording));
+                    forceLocationBtn.setEnabled(true);
+                } else {
+
+                    // Get the Tracks recorded
+                    List<Location> tracks = kujakuMapView.stopTrackingService(getApplicationContext());
```

Comment:
For some reason this does not seem to work now when you go back to the tracking activity by clicking on the notification. When the activity reopens and you click on **Stop Recording** the notification is still available and clicking on it again will still reopen the activity with the **Stop Recording** button still available.

**STEPS TO REPRODUCE**

1. Start tracking
2. Go home or to another activity
3. Click on the tracking notification which opens the tracking activity
4. Click on **Stop Recording**, the button changes to **START RECORDING** but the notification is still available and can be clicked again to reopen the activity. When reopened from the notification, it shows **STOP RECORDING**

Reply:
This has been fixed